testing/go/dolt_tables_test.go

```diff
@@ -1874,21 +1873,122 @@ func TestUserSpaceDoltTables(t *testing.T) {
 					},
 				},
 				{
-					Query:    "update dolt_rebase set action='reword', commit_message='insert rows' where rebase_order=1;",
+					Query: "select rebase.commit_message from dolt.rebase order by rebase_order;",
+					Expected: []sql.Row{
+						{"inserting row 1"},
+						{"inserting row 2"},
+						{"inserting row 3"},
+					},
+				},
+				{
+					Skip:  true, // TODO: table not found: dolt_rebase
```

Comment:
Shouldn't this work with your other PR?

Reply:
No the `dolt_x` table name identifier for a column for tables we converted to `dolt.x` doesn't work. Still need to look into that more